components/Tabs2/src/private/MDCTabBarViewItemView.m

```diff
@@ -23,8 +23,82 @@
 /** The minimum height of any item view. */
 static const CGFloat kMinHeight = 48;
 
+/// Outer edge padding from spec: https://material.io/go/design-tabs#spec.
+static const UIEdgeInsets kEdgeInsets = {.top = 12, .right = 16, .bottom = 12, .left = 16};
```

Comment:
Please make these configurable on the view (or add a TODO comment).

Reply:
I am not sure what do you mean. Let's sync this as well in the GVC.

Reply:
We can probably skip it for now, but in a follow-up we should expect to provide an API like `contentEdgeInsets` to manipulate the internal spacing of the content from the bounds.